Source/UIViewController+STZPopupView.swift

```diff
@@ -98,30 +98,30 @@ extension UIViewController {
         showAnimation()
     }
 
-    private func showAnimation() {
+    fileprivate func showAnimation() {
         if let config = config {
             switch (config.showAnimation) {
-            case .None:
+            case .none:
                 completionShowAnimation(true)
-            case .FadeIn:
+            case .fadeIn:
                 fadeIn()
-            case .SlideInFromTop:
+            case .slideInFromTop:
                 slideInFromTop()
-            case .SlideInFromBottom:
+            case .slideInFromBottom:
                 slideInFromBottom()
-            case .SlideInFromLeft:
+            case .slideInFromLeft:
                 slideInFromLeft()
-            case .SlideInFromRight:
+            case .slideInFromRight:
                 slideInFromRight()
-            case .Custom:
+            case .custom:
                 if let containerView = containerView, let popupView = popupView {
                     config.showCustomAnimation(containerView, popupView, { self.completionShowAnimation(true) })
                 }
             }
         }
     }
 
-    private func completionShowAnimation(finished: Bool) {
+    fileprivate func completionShowAnimation(_ finished: Bool) {
```

Comment:
I prefer not to use underscore.
`private func completionShowAnimation(finished: Bool)`